closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -344,23 +344,29 @@ private static void getProfileID(Properties props) {
                         + selectedProfile);
             }
             String defaultMatch = null;
+            Boolean profileExist = false;
```

Comment:
This should use the primitive type `boolean` instead of the wrapper type. A better name is `profileExists`.